specification.markdown

```diff
@@ -386,41 +386,60 @@ The path defines the allowed storage locations of the EBML Element within an EBM
+The EBMLAtomName of the EBMLElement part MUST be equal to the `@name` attribute of the EBML Schema.
+If the EBMLElement part contains an IsRecursive part, the EBML Element can occur within itself recursively (see (#recursive)).
 
-The starting PathDelimiter of the path corresponds to the root of the EBML Document.
+The starting PathDelimiter of EBMLParentPath corresponds to the root of the EBML Document.
 
-In some cases the EBMLLastParent part of the path is an EBMLGlobalParent. A path with a EBMLGlobalParent defines a Global Element; see (#global-elements). Any path that starts with the EBMLFixedParent of the Global Element and matches the occurrences found in the GlobalParentOccurence is a valid path for the Global Element. 
+The `@path` value MUST be unique within the EBML Schema. The `@id` value corresponding to this `@path` MUST NOT be defined for use within another EBML Element with the same EBMLParentPath as this `@path`.
 
-The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition. The repetition amounts correspond to the amount of unspecified Parent Element levels there can be between the EBMLFixedParent and the actual EBMLElementPath.
+A path with a GlobalPlaceholder as the EBMLLastParent defines a Global Element; see (#global-elements).
+If the element has no EBMLLastParent part or the EBMLLastParent part is not a GlobalPlaceholder then the Element is not a Global Element.
 
-PathMinOccurrence represents the minimum number of element path required between the EBMLFixedParent and the Global Element EBMLElementPath. For example 0 means the EBMLElementPath can be right after the EBMLFixedParent, 1 means there has to be at least an element between the EBMLFixedParent and the EBMLElementPath. If PathMinOccurrence is not present then that EBML Element has an PathMinOccurrence value of 0.
+The GlobalParentOccurence part is interpreted as the amount of valid EBMLPathAtom parts that can replace the GlobalPlaceholder in the path.
+PathMinOccurrence represents the minimum amount of EBMLPathAtom required to replace the GlobalPlaceholder.
+PathMaxOccurrence represents the maximum amount of EBMLPathAtom possible to replace the GlobalPlaceholder.
 
-PathMaxOccurrence represents the maximum number of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath. It cannot have the value 0 as it would be the Global Element can only be found right after the EBMLFixedParent, in which case it's not a Global Element anymore. If PathMaxOccurrence is not present then there is no upper bound for the permitted number of occurrences of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath.
+If PathMinOccurrence is not present then that GlobalParentOccurence has a PathMinOccurrence value of 0.
+If PathMaxOccurrence is not present then there is no upper bound for the permitted amount of EBMLPathAtom possible to replace the GlobalPlaceholder.
```

Comment:
I think we could use a statement, like `PathMaxOccurrence MUST be greater than PathMinOccurrence.`

Reply:
Yeah I left it out because it seems obvious but why not.
It can't be done in the ABNF itself ;(